### ISSUE TITLE
IPV6 seems to crash my Teaspeak server

### ISSUE THREAD

**Issue by NotYourMajesty** [TeaSpeak/TeaSpeak#550]
**Describe the bug**
Each time i have a player with an IPV6, the server crash. Strange thing :o

**Logs**
```
2019-01-18 05:05:21] [INFO]      2 | Voice client 217/xCgUTBAkf8HrvFTRluI9c1rh69k= (Fabien/Quentin Delarue) from 2a01:cb05:8b5e:400:ccf:7aa2:5df7:9a44:64322 joined.
[2019-01-18 05:05:21] [ERROR]     2 | Command handling of command clientupdate needs 81ms
[2019-01-18 05:05:21] [ERROR]     2 | [2a01:cb05:8b5e:400:ccf:7aa2:5df7:9a44:64322/Fabien/Quentin Delarue | 2] Handling of packet Command needs more than 10ms (81ms)
[2019-01-18 05:05:22] [ERROR]     2 | Command handling of command channelsubscribeall needs 30ms
[2019-01-18 05:05:22] [ERROR]     2 | [2a01:cb05:8b5e:400:ccf:7aa2:5df7:9a44:64322/Fabien/Quentin Delarue | 2] Handling of packet Command needs more than 10ms (30ms)
[2019-01-18 05:05:22] [[1m[7m[0;31mCRITICAL] [1m[7m[0;31m    2 | Failed to write datagram packet for client 2a01:cb05:8b5e:400:ccf:7aa2:5df7:9a44:64322 (errno: 97 message: Address family not supported by protocol). Closing connection[0m
[2019-01-18 05:05:22] [[1m[7m[0;31mCRITICAL] [1m[7m[0;31m    2 | Failed to write datagram packet for client 77.136.40.105:62434 (errno: 101 message: Network is unreachable). Closing connection[0m
[2019-01-18 05:05:22] [DEBUG]     2 | [2a01:cb05:8b5e:400:ccf:7aa2:5df7:9a44:64322/Fabien/Quentin Delarue | 2] Closing voice client connection. (Flush: false)
[2019-01-18 05:05:22] [DEBUG]     2 | [77.136.40.105:62434/Yozu | 1] Closing voice client connection. (Flush: false)
[2019-01-18 05:05:22] [INFO]      2 | Voice client 217/xCgUTBAkf8HrvFTRluI9c1rh69k= (Fabien/Quentin Delarue) from 2a01:cb05:8b5e:400:ccf:7aa2:5df7:9a44:64322 left.
[2019-01-18 05:05:22] [INFO]      2 | Voice client 2/Rq0TxFQCyxe9+zw2Ze7YGtPbJ5c= (Yozu) from 77.136.40.105:62434 left.
[2019-01-18 05:05:22] [DEBUG]     2 |  Deleting VoiceClient instance at 0x7f3496211010
[2019-01-18 05:05:22] [DEBUG]     2 |  Deleting VoiceClient instance at 0x7f3496211610
```

**Environment information:**
 Operating system: Ubuntu Server 18.04 "Bionic Beaver" LTS (64bits) 
 TeaSpeak version: Last Version
 I am using `experimental_31`: yes

**Comment by WolverinDEV** [comment]
Do you have a crash dump?
Because just this section lookes only like, that the user gets kicked because of a protocol error.

**Comment by WolverinDEV** [comment]
should fixed in 1.3.1-1 :)
